src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -74,7 +74,7 @@ const excludedForms = new Set([
   '24-0296',
   '10-10CG-example',
   '686C-674', // Until we import the schema
-  VA_FORM_IDS.FORM_21_526EZ, // old
+  VA_FORM_IDS.FORM_21_526EZ, // old (v1 removed)
```

Comment:
Since this is no longer part of `vets-json-schema` can this line just be removed? If so, we could also remove the entry in the `platform/forms/constants` file.

Reply:
We actually still use `FORM_21_526EZ` for v2

I'm not sure why all-claims is mapped to `21-526EZ-ALLCLAIMS` in this file, but we don't use it any where else (except in the vets-json-schema file name)

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/all-claims/config/form.js#L130

Reply:
Interesting, good to know. I'll go ahead and approve.